commit 7c01a8e56bf8a03b9b50a770de36abfee03c4fc5
Author: Christian Borntraeger <borntraeger@linux.ibm.com>
Date:   Tue Nov 15 02:31:13 2011 +0000

    qeth: remove WARN_ON leftover
    
    The patch "qeth: exploit asynchronous delivery of storage blocks"
    added a WARN_ON in qeth_schedule_recovery. A device recovery should
    not cause a kernel warning. This is obviously a debugging  left-over
    that we forgot to remove.
    
    Signed-off-by: Christian Borntraeger <borntraeger@de.ibm.com>
    Signed-off-by: Frank Blaschka <frank.blaschka@de.ibm.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

diff --git a/drivers/s390/net/qeth_core_main.c b/drivers/s390/net/qeth_core_main.c
index 81534437373a..fff57de78943 100644
--- a/drivers/s390/net/qeth_core_main.c
+++ b/drivers/s390/net/qeth_core_main.c
@@ -881,7 +881,6 @@ EXPORT_SYMBOL_GPL(qeth_do_run_thread);
 void qeth_schedule_recovery(struct qeth_card *card)
 {
 	QETH_CARD_TEXT(card, 2, "startrec");
-	WARN_ON(1);
 	if (qeth_set_thread_start_bit(card, QETH_RECOVER_THREAD) == 0)
 		schedule_work(&card->kernel_thread_starter);
 }